Cargo.toml

```diff
@@ -51,15 +51,15 @@ external-secp = []
 # All these dependencies must match the versions in:
 # https://github.com/zcash/zcash/blob/<git subtree version>/Cargo.toml
 blake2b_simd = "1"
-cxx = { version = "=1.0.79", features = ["c++17"] }
+cxx = { version = "=1.0.83", features = ["c++17"] }
 libc = "0.2"
 memuse = "0.2"
 orchard = "0.3"
 rand_core = "0.6"
 tracing = "0.1"
 zcash_encoding = "0.2"
 zcash_note_encryption = "0.2"
-zcash_primitives = { version = "0.8.1", features = ["transparent-inputs"] }
+zcash_primitives = { version = "0.9.1", features = ["transparent-inputs"] }
```

Comment:
I checked that these match https://github.com/zcash/zcash/blob/774f2d8eda62cc51990e0aa154f92f09ffba3873/Cargo.toml

However a heads up:  `zebra` is using `zcash_proofs 0.10.0` (in `zebra-consensus`) which uses `zcash_primitives 0.10.0` which will cause a duplicated dependency. Not sure how we'll want to solve that

Reply:
> Not sure how we'll want to solve that

It's still using some methods that were removed in v0.10.0 so we're stuck with an item in zebra's deny.toml for now.